addon/components/em-select.js

```diff
@@ -99,7 +99,7 @@ export default Component.extend(InputComponentMixin, {
         if(selectedIndex !== 0){
           selectedIndex--;
         } else {
-          if (this.get('property')) {
+          if (this.get(this.get('property'))) {
```

Comment:
Have you tried this change? Because I think it should check on model if `this.get('property')` is set.

Reply:
Sorry, I was trying to patch the issue that I just realised was fixed in 570dffd6f59f6bf7f290e700e445a58af422fea6.
Before that commit, it was `if (model.get('property')) {` and I fixed it locally by changing it to `if (model.get(this.get('property'))) {` thinking that was the intended check.
Then I didn't notice I was applying my fix to 570dffd6f59f6bf7f290e700e445a58af422fea6 which was fixed better anyway.

Reply:
I have pointed my project to the master branch until that commit is released.

Reply:
Okay. Thanks for making this PR anyway. Gonna release that version soon. Hopefully with the other PR merged.